diff --git a/french/common.rpy b/tl/french/common.rpy
similarity index 99%
rename from french/common.rpy
rename to tl/french/common.rpy
index ca8abfc..a9a914b 100644
--- a/french/common.rpy
+++ b/tl/french/common.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 translate french strings:
 
diff --git a/french/day01.rpy b/tl/french/day01.rpy
similarity index 99%
rename from french/day01.rpy
rename to tl/french/day01.rpy
index 334064b..bc8ab6c 100644
--- a/french/day01.rpy
+++ b/tl/french/day01.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day01.rpy:6
 translate french day01_cf4be568:
diff --git a/french/day02.rpy b/tl/french/day02.rpy
similarity index 99%
rename from french/day02.rpy
rename to tl/french/day02.rpy
index 1104944..baa48b6 100644
--- a/french/day02.rpy
+++ b/tl/french/day02.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day02.rpy:8
 translate french day02_07e48f6f:
diff --git a/french/day03.rpy b/tl/french/day03.rpy
similarity index 99%
rename from french/day03.rpy
rename to tl/french/day03.rpy
index aa3c720..a0e6359 100644
--- a/french/day03.rpy
+++ b/tl/french/day03.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day03.rpy:13
 translate french day03_b8f4b6b0:
diff --git a/french/day04.rpy b/tl/french/day04.rpy
similarity index 99%
rename from french/day04.rpy
rename to tl/french/day04.rpy
index 38cffdd..6ca67c0 100644
--- a/french/day04.rpy
+++ b/tl/french/day04.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day04.rpy:19
 translate french day04_716cca4a:
diff --git a/french/day05.rpy b/tl/french/day05.rpy
similarity index 99%
rename from french/day05.rpy
rename to tl/french/day05.rpy
index 0b53a9c..7472f40 100644
--- a/french/day05.rpy
+++ b/tl/french/day05.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day05.rpy:25
 translate french day05_43a3b4e7:
diff --git a/french/day06.rpy b/tl/french/day06.rpy
similarity index 99%
rename from french/day06.rpy
rename to tl/french/day06.rpy
index c54df1e..68887b4 100644
--- a/french/day06.rpy
+++ b/tl/french/day06.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day06.rpy:16
 translate french day06_06623250:
diff --git a/french/day07.rpy b/tl/french/day07.rpy
similarity index 99%
rename from french/day07.rpy
rename to tl/french/day07.rpy
index 044cdb9..5d770b2 100644
--- a/french/day07.rpy
+++ b/tl/french/day07.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day07.rpy:21
 translate french day07_0dc2d14d:
diff --git a/french/day08.rpy b/tl/french/day08.rpy
similarity index 99%
rename from french/day08.rpy
rename to tl/french/day08.rpy
index 1081370..6befc44 100644
--- a/french/day08.rpy
+++ b/tl/french/day08.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day08.rpy:19
 translate french day08_8a8c5fc0:
diff --git a/french/day09.rpy b/tl/french/day09.rpy
similarity index 99%
rename from french/day09.rpy
rename to tl/french/day09.rpy
index 9878e57..074d260 100644
--- a/french/day09.rpy
+++ b/tl/french/day09.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day09.rpy:27
 translate french day09_4c4ab8d0:
diff --git a/french/day10.rpy b/tl/french/day10.rpy
similarity index 99%
rename from french/day10.rpy
rename to tl/french/day10.rpy
index 486c5d1..348db5f 100644
--- a/french/day10.rpy
+++ b/tl/french/day10.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day10.rpy:18
 translate french day10_1c7cf5ff:
diff --git a/french/day11.rpy b/tl/french/day11.rpy
similarity index 99%
rename from french/day11.rpy
rename to tl/french/day11.rpy
index 089b25d..8e01222 100644
--- a/french/day11.rpy
+++ b/tl/french/day11.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day11.rpy:25
 translate french day11_f2bcf4e4:
diff --git a/french/day12.rpy b/tl/french/day12.rpy
similarity index 99%
rename from french/day12.rpy
rename to tl/french/day12.rpy
index 75565ef..19e3887 100644
--- a/french/day12.rpy
+++ b/tl/french/day12.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day12.rpy:14
 translate french day12_484f2a11:
diff --git a/french/day13.rpy b/tl/french/day13.rpy
similarity index 99%
rename from french/day13.rpy
rename to tl/french/day13.rpy
index 2f4879c..a5b292b 100644
--- a/french/day13.rpy
+++ b/tl/french/day13.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day13.rpy:24
 translate french day13_05c585c9:
diff --git a/french/day14.rpy b/tl/french/day14.rpy
similarity index 99%
rename from french/day14.rpy
rename to tl/french/day14.rpy
index c3eba9c..d6733a2 100644
--- a/french/day14.rpy
+++ b/tl/french/day14.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day14.rpy:18
 translate french day14_dc7d7674:
diff --git a/french/day15.rpy b/tl/french/day15.rpy
similarity index 99%
rename from french/day15.rpy
rename to tl/french/day15.rpy
index 0dd9ac2..f2eb956 100644
--- a/french/day15.rpy
+++ b/tl/french/day15.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day15.rpy:21
 translate french day15_f0bda93d:
diff --git a/french/day16.rpy b/tl/french/day16.rpy
similarity index 99%
rename from french/day16.rpy
rename to tl/french/day16.rpy
index 49b2062..b8b650c 100644
--- a/french/day16.rpy
+++ b/tl/french/day16.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day16.rpy:26
 translate french day16_8e90f4f7:
diff --git a/french/day17.rpy b/tl/french/day17.rpy
similarity index 99%
rename from french/day17.rpy
rename to tl/french/day17.rpy
index 69bbe77..805b0aa 100644
--- a/french/day17.rpy
+++ b/tl/french/day17.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day17.rpy:19
 translate french day17_a85bc899:
diff --git a/french/day18.rpy b/tl/french/day18.rpy
similarity index 99%
rename from french/day18.rpy
rename to tl/french/day18.rpy
index a7145bc..b42472d 100644
--- a/french/day18.rpy
+++ b/tl/french/day18.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day18.rpy:18
 translate french day18_06be7dc0:
diff --git a/french/day19.rpy b/tl/french/day19.rpy
similarity index 99%
rename from french/day19.rpy
rename to tl/french/day19.rpy
index d387aba..9a86cfd 100644
--- a/french/day19.rpy
+++ b/tl/french/day19.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day19.rpy:18
 translate french day19_39644161:
diff --git a/french/day20.rpy b/tl/french/day20.rpy
similarity index 99%
rename from french/day20.rpy
rename to tl/french/day20.rpy
index 209e6c2..6d126db 100644
--- a/french/day20.rpy
+++ b/tl/french/day20.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day20.rpy:23
 translate french day20_46f107d0:
diff --git a/french/day21.rpy b/tl/french/day21.rpy
similarity index 99%
rename from french/day21.rpy
rename to tl/french/day21.rpy
index 1577941..3f71f44 100644
--- a/french/day21.rpy
+++ b/tl/french/day21.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day21.rpy:16
 translate french day21_12f2210b:
diff --git a/french/day22.rpy b/tl/french/day22.rpy
similarity index 99%
rename from french/day22.rpy
rename to tl/french/day22.rpy
index 8bfc376..c43148b 100644
--- a/french/day22.rpy
+++ b/tl/french/day22.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day22.rpy:23
 translate french day22_b37a99fa:
diff --git a/french/day23.rpy b/tl/french/day23.rpy
similarity index 99%
rename from french/day23.rpy
rename to tl/french/day23.rpy
index 8743478..2827c49 100644
--- a/french/day23.rpy
+++ b/tl/french/day23.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day23.rpy:42
 translate french day23_58fe2451:
diff --git a/french/day24.rpy b/tl/french/day24.rpy
similarity index 99%
rename from french/day24.rpy
rename to tl/french/day24.rpy
index 73a3f83..e99664f 100644
--- a/french/day24.rpy
+++ b/tl/french/day24.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day24.rpy:15
 translate french day24_0cf04f96:
diff --git a/french/day25.rpy b/tl/french/day25.rpy
similarity index 99%
rename from french/day25.rpy
rename to tl/french/day25.rpy
index dd119cc..d1e97cf 100644
--- a/french/day25.rpy
+++ b/tl/french/day25.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day25.rpy:16
 translate french day25_49838de2:
diff --git a/french/day26.rpy b/tl/french/day26.rpy
similarity index 99%
rename from french/day26.rpy
rename to tl/french/day26.rpy
index bc3094a..e588d54 100644
--- a/french/day26.rpy
+++ b/tl/french/day26.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day26.rpy:23
 translate french day26_aa327362:
diff --git a/french/day27.rpy b/tl/french/day27.rpy
similarity index 99%
rename from french/day27.rpy
rename to tl/french/day27.rpy
index 3ac8f41..e6b1891 100644
--- a/french/day27.rpy
+++ b/tl/french/day27.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day27.rpy:22
 translate french day27_7cd10344:
diff --git a/french/day28.rpy b/tl/french/day28.rpy
similarity index 99%
rename from french/day28.rpy
rename to tl/french/day28.rpy
index e063d08..c3bb5d4 100644
--- a/french/day28.rpy
+++ b/tl/french/day28.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day28.rpy:22
 translate french day28_d8fd40c4:
diff --git a/french/day29.rpy b/tl/french/day29.rpy
similarity index 99%
rename from french/day29.rpy
rename to tl/french/day29.rpy
index c62b5ad..07d6448 100644
--- a/french/day29.rpy
+++ b/tl/french/day29.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day29.rpy:12
 translate french day29_a0fb2736:
diff --git a/french/day30.rpy b/tl/french/day30.rpy
similarity index 99%
rename from french/day30.rpy
rename to tl/french/day30.rpy
index feb88ec..b334e5f 100644
--- a/french/day30.rpy
+++ b/tl/french/day30.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day30.rpy:16
 translate french day30_965f6063:
diff --git a/french/day31.rpy b/tl/french/day31.rpy
similarity index 99%
rename from french/day31.rpy
rename to tl/french/day31.rpy
index 789489d..c9d5500 100644
--- a/french/day31.rpy
+++ b/tl/french/day31.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day31.rpy:30
 translate french day31_82f9b460:
diff --git a/french/day32.rpy b/tl/french/day32.rpy
similarity index 99%
rename from french/day32.rpy
rename to tl/french/day32.rpy
index c35e1d1..4e200ba 100644
--- a/french/day32.rpy
+++ b/tl/french/day32.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day32.rpy:16
 translate french day32_53456d3d:
diff --git a/french/day33.rpy b/tl/french/day33.rpy
similarity index 99%
rename from french/day33.rpy
rename to tl/french/day33.rpy
index a55814c..de5a6d1 100644
--- a/french/day33.rpy
+++ b/tl/french/day33.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day33.rpy:20
 translate french day33_f03211b0:
diff --git a/french/day34.rpy b/tl/french/day34.rpy
similarity index 99%
rename from french/day34.rpy
rename to tl/french/day34.rpy
index 3fcfb8e..d297056 100644
--- a/french/day34.rpy
+++ b/tl/french/day34.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/day34.rpy:15
 translate french day34_5a640ca6:
diff --git a/french/extras.rpy b/tl/french/extras.rpy
similarity index 99%
rename from french/extras.rpy
rename to tl/french/extras.rpy
index f314405..6866140 100644
--- a/french/extras.rpy
+++ b/tl/french/extras.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/extras.rpy:10
 translate french extra_scene_01_3eb4f371:
diff --git a/french/options.rpy b/tl/french/options.rpy
similarity index 59%
rename from french/options.rpy
rename to tl/french/options.rpy
index 0b73faf..6e42a1d 100644
--- a/french/options.rpy
+++ b/tl/french/options.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 translate french strings:
 
diff --git a/french/screens.rpy b/tl/french/screens.rpy
similarity index 99%
rename from french/screens.rpy
rename to tl/french/screens.rpy
index 82ceac4..1ae395e 100644
--- a/french/screens.rpy
+++ b/tl/french/screens.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 translate french strings:
 
diff --git a/french/script.rpy b/tl/french/script.rpy
similarity index 99%
rename from french/script.rpy
rename to tl/french/script.rpy
index d87fa5a..464255f 100644
--- a/french/script.rpy
+++ b/tl/french/script.rpy
@@ -1,4 +1,4 @@
-﻿# TODO: Translation updated at 2019-02-22 21:58
+﻿# TODO: Translation updated at 2019-02-22 22:04
 
 # game/script.rpy:123
 translate french start_9f2b9dde:
